e2e/install_test.go

```diff
@@ -833,11 +797,9 @@ func TestResetAndReinstallAirgap(t *testing.T) {
 	t.Logf("%s: waiting for nodes to reboot", time.Now().Format(time.RFC3339))
 	time.Sleep(30 * time.Second)
 
-	t.Logf("%s: installing embedded-cluster on node 0", time.Now().Format(time.RFC3339))
-	line = []string{"single-node-airgap-install.sh", os.Getenv("SHORT_SHA")}
-	if _, _, err := tc.RunCommandOnNode(0, line); err != nil {
-		t.Fatalf("fail to install embedded-cluster on node %s: %v", tc.Nodes[0], err)
-	}
+	installSingleNodeWithOptions(t, tc, installOptions{
+		networkInterface: "tailscale0",
+	})
```

Comment:
why is this not also airgap?

Reply:
If this passed _without_ the airgap flag I have questions

Reply:
Whooh, how did this pass...